src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -60,14 +71,31 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 			target = localize('executionPlanPropertiesEdgeOperationName', "Edge");
 		}
 
-		const titleText = localize('executionPlanComparisonPropertiesBottomOperation', "Bottom operation: {0}", target);
-		this._bottomOperationNameContainer.innerText = titleText;
-		this._bottomOperationNameContainer.title = titleText;
+		this._rightTitleText = localize('executionPlanComparisonPropertiesRightOperation', "Right operation: {0}", target);
+		this._bottomTitleText = localize('executionPlanComparisonPropertiesBottomOperation', "Bottom operation: {0}", target);
+		this._bottomOperationNameContainer.innerText = this._bottomTitleText;
+		this._bottomOperationNameContainer.title = this._bottomTitleText;
 		this.addDataToTable();
 	}
 
+	public updatePropertyContainerTitles(orientation: 'horizontal' | 'vertical'): void {
+		if (orientation === 'horizontal') {
+			this._topOperationNameContainer.innerText = this._topTitleText;
```

Comment:
Having the container named "top" be either on the top or left and the one named "bottom" be either on the right or bottom isn't very intuitive. I'd suggest using a more generic name for these - maybe something like `primary` and `secondary` like the side by side editor uses.

Reply:
I'm ok with being consistent based on precedent set elsewhere in ADS.  If that means primary and secondary, that's fine, as long as it's the same throughout the app.

Reply:
To be clear - I'm mostly talking about the internal property names. Having top/bottom left/right displayed to the user makes sense to me.

Reply:
Got it!  Thanks for clarifying :)